### ISSUE TITLE
Invalid filename chars in plot pngs

### ISSUE THREAD

**Issue by boulund** [will-rowe/groot#15]
Recently encountered this in groot 0.8.3 from conda, using the groot-db:

```
panic: open ./groot-plots/coverage-for-groot-db_RESFINDER__tet(0/32/0)_7_FP929050.png: no such file or directory          
                                                                                                                          
goroutine 1 [running]:                                                                                                    
github.com/will-rowe/groot/src/reporting.(*BAMreader).Run(0xc4201bc900)                                                   
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/will-rowe/groot/src/reporting/reporting.go:191 +0xf7c
github.com/will-rowe/groot/cmd.runReport()                                                                                
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/will-rowe/groot/cmd/report.go:140 +0x2a3             
github.com/will-rowe/groot/cmd.glob..func7(0x1036140, 0xc4201dc090, 0x0, 0x9)                                             
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/will-rowe/groot/cmd/report.go:56 +0x20               
github.com/spf13/cobra.(*Command).execute(0x1036140, 0xc4201dc000, 0x9, 0x9, 0x1036140, 0xc4201dc000)                     
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/spf13/cobra/command.go:766 +0x2c1                    
github.com/spf13/cobra.(*Command).ExecuteC(0x10363a0, 0x1, 0x1, 0x1036600)                                                
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/spf13/cobra/command.go:852 +0x30a                    
github.com/spf13/cobra.(*Command).Execute(0x10363a0, 0xc420032478, 0x0)                                                   
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/spf13/cobra/command.go:800 +0x2b                     
github.com/will-rowe/groot/cmd.Execute()                                                                                  
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/will-rowe/groot/cmd/root.go:64 +0x2d                 
main.main()                                                                                                               
        /opt/conda/conda-bld/groot_1540475137626/work/src/github.com/will-rowe/groot/main.go:26 +0x20                     
```

I think it's due to the forward slashes in the filename it attempts to write to.

**Comment by will-rowe** [comment]
Thanks for this - looks like you're right. I will fix this and make a new release.

**Comment by will-rowe** [comment]
Thanks again for catching this. I have fixed it in 478fa64f2b2eb73bda301613d128d2c6b409c3f0.

Will tag a new release and send v0.8.4 to conda.

**Comment by will-rowe** [comment]
It's in conda now. Closing issue.
